test/sanity/run-test.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Copyright 2022 The Ceph CSI Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function cleanup {
+  echo 'pkill -f cephcsi'
+  if [ -z "$GITHUB_ACTIONS" ]
```

Comment:
use `${GITHUB_ACTIONS}` formatting of variables

The GitHub Workflow is missing for this?